pkg/kubelet/kubelet_network.go

```diff
@@ -137,10 +127,6 @@ func parseResolvConf(reader io.Reader, dnsScrubber dnsScrubber) (nameservers []s
 		}
 	}
 
-	// Give the cloud-provider a chance to post-process DNS settings.
```

Comment:
leave me a TODO or at least a comment saying that this used to scrub DNS for each cloud, but that doesn't make sense once cloud-provider is factorerd out.